### PR TITLE
added works subdomain and some records for vercel

### PR DESCRIPTION
**Pull request by ArnavBarway**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

[works-gamma-beryl.vercel.app](https://works-gamma-beryl.vercel.app/)

### CODE REVIEW

**Review by ArnavBarway — COMMENTED** [review]
done

**Review by ArnavBarway — COMMENTED** [review]
done

**Review by wdhdev — APPROVED** [review]

File: domains/_vercel.works.arnxv.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "ArnavBarway",
+    "email": "arnav@sagaraga.com"
+  },
+  "record": [
+    {
+      "TXT": "vc-domain-verify=works.arnxv.is-a.dev,f8b0553cc89bfe7097fa"
+    }
+  ]
```
Comment (wdhdev) [inline domains/_vercel.works.arnxv.json:10]:
```suggestion
  "record": {
      "TXT": "vc-domain-verify=works.arnxv.is-a.dev,f8b0553cc89bfe7097fa"
   }
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by ArnavBarway** [comment]
thanks for suggesting

**Comment by ArnavBarway** [comment]
@wdhdev

**Review by wdhdev — APPROVED** [review]

**Comment by ArnavBarway** [comment]
thx

**Pull request merged**